scripts/parallel.js

```diff
@@ -0,0 +1,86 @@
+const webdriver = require('selenium-webdriver');
```

Comment:
general comment regarding coding practices in the repo, let's have proper indentation and follow 80 column width for each row. Also either have `;` in all places or not. Also empty line at EoF for all files.